data/pug.txt

```diff
@@ -192,6 +193,7 @@ outfit "Pug Gridfire Turret"
 		"hull damage" 600
 		"blast radius" 40
 		"piercing" .3
+	description "The Gridfire Turret is a very powerful weapon, designed to disable (or as well destroy) even the strongest of the warships."
```

Comment:
Seems unrelated to this patch.

Reply:
Are you sure? The other outfits, if they have an image, have got a description as well

Reply:
Argh, sorry, I saw the diff against the previous commit and thought you were removing text that was already there. Ignore me.

Reply:
It's still quite a bland description, can anyone come up with something better? How about `No one quite knows how the Gridfire Turret works, and the Pug aren't interested explaining it. But it is clearly capable of releasing tremendous energies at any point in space to dice even the strongest of warships.`.
Even if you (probably sensibly) ignore me you should remove the last `...the...` and the `...as well...` for the sake of reading well.

Reply:
I'll use yours, wait about 6 hours cuz I don't have time right now